src/solve_dc_powerflow.jl

```diff
@@ -114,3 +201,70 @@ function solve_powerflow(
     solve_powerflow!(data)
     return write_results(data, sys)
 end
+
+# MULTI PERIOD ###############################################################
+
+"""
+Evaluates the power flows on each system's branch by means of the PTDF matrix.
+Updates the PowerFlowData structure "data" and returns a dictionary containing
+a number of DataFrames equal to the numeber of timestep considered in "data".
+Each DataFrame containts the flows and angles.
+
+# Arguments:
+- `data::PTDFPowerFlowData`:
+        PowerFlowData structure containing the system's data per each timestep
+        considered, as well as the PTDF matrix.
+- `sys::PSY.System`:
+        container gathering the system data.
+"""
+function solve_powerflow(
```

Comment:
```suggestion
function solve_powerflow!(
```

Reply:
I have defined two function, one that overwrites the input (solve_powerflows!), and one that returns a dictionary of data frames (solve_powerflows). It is true though that "solve_powerflows" calls "solve_powerflows!".

shouldn't be better to name the function writing a dictionary of dataframes with no "!" ?